### DISCUSSION TITLE
zh/nixos-with-flakes/modularize-the-configuration

### DISCUSSION THREAD

**Discussion by giscus** [ryan4yin/nixos-and-flakes-book#160] [Announcements]
# zh/nixos-with-flakes/modularize-the-configuration

一份非官方的新手指南

https://nixos-and-flakes.thiscute.world/zh/nixos-with-flakes/modularize-the-configuration

**Comment by lVentus** [comment]
想问一下，能在HM中配置的是不是只有programs或者services下安装的软件啊。通过home.packages = with pkgs安装的是不是只能连接点文件啊。

**Reply by ryan4yin** [reply]
`home.packages` 是安装软件，`programs` 是安装软件与其配置文件，`services` 是安装并运行 user-level 的 systemd 服务。
